createlinks

```diff
@@ -7,6 +7,7 @@ my $event = 'nethserver-collabora-update';
 event_actions($event, qw(
              initialize-default-databases 00
              nethserver-collabora-conf 20
+             nethserver-collabora-conf-php 80
```

Comment:
in fact I think we should trigger this event at 99, just to be sure it is the end

Reply:
I said something wrong @mrmarkuz , in fact we have to set it to 99 because it is triggered after the runlevel-adjust (service restart)

Thnak for your change, we go to testing again.